agent/src/com/jonnyzzz/teamcity/virtual/run/docker/DockerVM.java

```diff
@@ -147,7 +163,7 @@ public BuildProcess startImpl() throws RunBuildException {
                 "--name=" + name,
                 "-v",
                 checkoutDir.getPath() + ":" + mountPoint + ":" + ctx.getDockerMountMode(),
-                "--workdir=" + mountPoint + "/" + RelativePaths.resolveRelativePath(checkoutDir, workDir),
+                "--workdir=" + mountPoint + ctx.getPathSeparatorInsideContainer() + RelativePaths.resolveRelativePath(checkoutDir, workDir),
```

Comment:
It may easily turn out we run on Linux to start remote container on Windows. We cannot relay on slashes returned from the command